### ISSUE TITLE
Advisories should be able to be commented

### ISSUE THREAD

**Issue by JanHoefelmeyer** [ISDuBA/ISDuBA#42]
When looking at an advisory, currently, you can write a comment within the designated text area, but pressing the send button has no effect. This seems to be a bug.

**Comment by s-l-teichmann** [comment]
Is the Advisory in 'read' or 'assessing' state? Only these are allowed to be commented.

**Comment by JanHoefelmeyer** [comment]
Assuming that an advisory is properly set into the "read" state when switching the workflow trigger on the top left to "read" (from new), then yes. It was in the read state. If that selection is currently merely cosmetic and something else has to be done to properly set the state, then that may be the issue.

Although switching the workflow state may likely be the actually bugged process, considering the console proclaims an internal server error (500), and setting the state does indeed not persist between reloads.

**Comment by s-l-teichmann** [comment]
The internal server errors are fixed in the ssvc-calc branch which will soon be merged back to main.

**Comment by ThomasJunk** [comment]
Idependently: 

Commenting is currently broken.

Steps to reproduce:
1) Select random Advisory
2) Open its document
3) Enter a comment
4) Press send

Sending a request
```
http://localhost:5173/api/comments/undefined
```

response:

```
{
    "error": {
        "Func": "ParseInt",
        "Num": "undefined",
        "Err": {}
    }
}
```

**Comment by ThomasJunk** [comment]
@cintek Would you have a look what goes wrong here?

**Comment by JanHoefelmeyer** [comment]
While only a minor issue, additionally you can be visually told that you are within a text field overlapping the actual text field, which is not writable.  

I have it rarely be reproduced by clicking on the current send, but consistently reproduced by clicking at the bottom of the text field where the mouse-shape-preview stops showing that clicking would enter a text field.

**Comment by cintek** [comment]
Fixed by b62199719c3d438454bf78c69c7f11b1a0d72776. @JanHoefelmeyer can you check if it works now?

**Comment by JanHoefelmeyer** [comment]
Advisories are now able to be commented, and comments can be edited.

Comments persist as expected and are timestamped.
